src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-fri calendar--month-length-31">
```

Comment:
proper modifier names should be: `block-name--modifier-name--modifier-value`
please fix it